linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_deny_root/rule.yml

```diff
@@ -49,12 +49,29 @@ references:
     stigid@rhel7: RHEL-07-010330
     stigid@rhel8: RHEL-08-020022
 
-ocil_clause: 'that is not the case'
+ocil_clause: 'limiting the number of failed logon attempts for the root user is not configured'
 
 ocil: |-
     To ensure that even the <tt>root</tt> account is locked after a defined number of failed password
     attempts, run the following command:
     <pre>$ grep even_deny_root /etc/pam.d/system-auth</pre>
     The output should show <tt>even_deny_root</tt>.
 
+fix: |-
```

Comment:
Is the intention, then, to have `fix` be manual steps to remediate something? Can't we somehow derive this from the "Ansible" or "bash" remediations?

Reply:
> Is the intention, then, to have fix be manual steps to remediate something? 

Currently, yes

> Can't we somehow derive this from the "Ansible" or "bash" remediations?

Possibly. I believe that we will still need to this key in some cases. But for cases where there are bash remediations we might be able to use those.

Reply:
fair enough.